src/main/java/org/skriptlang/skript/bukkit/loottables/elements/expressions/ExprLootItems.java

```diff
@@ -0,0 +1,90 @@
+package org.skriptlang.skript.bukkit.loottables.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.loot.LootContext;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+@Name("Loot Table Items")
+@Description({
+	"Returns the items of a loot table using a loot context. " +
+		"Some loot tables will require some of these values whereas others may not."
+})
+@Examples({
+	"set {_items::*} to loot items of the loot table \"minecraft:chests/simple_dungeon\" with loot context {_context}",
+	"# this will set {_items::*} to the items that would be dropped from the simple dungeon loot table with the given loot context",
+	"",
+	"give player loot items of entity's loot table with loot context {_context}",
+	"# this will give the player the items that the entity would drop with the given loot context"
+})
+@Since("INSERT VERSION")
+public class ExprLootItems extends SimpleExpression<ItemStack> {
+
+	static {
+		Skript.registerExpression(ExprLootItems.class, ItemStack.class, ExpressionType.COMBINED,
+			"[the] loot [item[s]] of [[the] loot[ ]table[s]] %loottables% (with|using) [[the] [loot] context] %lootcontext%",
+			"[the] %loottables%'[s] loot [item[s]] (with|using) [[the] [loot] context] %lootcontext%"
```

Comment:
```suggestion
			"[the] (loot|item[s]) of [[the] loot[ ]table[s]] %loottables% (with|using) [[the] [loot] context] %lootcontext%",
			"[the] %loottables%'[s] (loot|item[s]) (with|using) [[the] [loot] context] %lootcontext%"
```

Reply:
i think loot and items are better off not seeing each other anymore

Reply:
I feel like items is too generic

Reply:
Hmm
Maybe just loot then, since i don't think loot items makes much grammatical sense